app/domain/authentication/authn_jwt/fetch_cached_signing_key.rb

```diff
@@ -0,0 +1,18 @@
+module Authentication
+  module AuthnJwt
+
+    FetchCachedSigningKey ||= CommandClass.new(
+      dependencies: {
+        fetch_singing_key_interface: ::Authentication::AuthnJwt::FetchSigningKeyInterface,
+        logger: Rails.logger
+      },
+      inputs: %i[]
+    ) do
+
+    def call
+      @fetch_singing_key_interface.fetch_signing_key
```

Comment:
Should this line be moved to another function so call only with call functions and don't do logic itself?